ultrasonic/src/main/kotlin/org/moire/ultrasonic/adapters/TrackViewHolder.kt

```diff
@@ -112,11 +115,15 @@ class TrackViewHolder(val view: View) : RecyclerView.ViewHolder(view), Checkable
             progress.isVisible = false
         }
 
-        RxBus.playerStateObservable.subscribe {
+        rxSubscription = RxBus.playerStateObservable.subscribe {
```

Comment:
I'm not sure how this worked without unsubscribing, but doing it explicitly feels safer